### PR TITLE
TST: Fix skipped unit tests in test_ga. Install python-gflags using p…

### LINKED ISSUE #11090: Unit tests in test_ga are being skipped
While working on another issue, I noticed the following error in the build test log on travis.
```
-------------------------------------------------------------
#15 nose.failure.Failure.runTest: need httplib2 and auth libs
-------------------------------------------------------------
```
I'm not familar with the ga module, but 'll try to fix it

### PR DESCRIPTION
**Pull request by parthea**
closes #11090 

This commit should resolve the following error in the travis build log.

```
-------------------------------------------------------------
#15 nose.failure.Failure.runTest: need httplib2 and auth libs
-------------------------------------------------------------
```

Valid Google credentials are required to run the ga unit tests.

### CODE REVIEW

**Comment by parthea** [comment]
Here is the output from my local test session:

```
test_getdata (pandas.io.tests.test_ga.TestGoogle) ... ok
test_iterator (pandas.io.tests.test_ga.TestGoogle) ... ok
test_remove_token_store (pandas.io.tests.test_ga.TestGoogle) ... ok
test_segment (pandas.io.tests.test_ga.TestGoogle) ... ok
test_v2_advanced_segment_format (pandas.io.tests.test_ga.TestGoogle) ... ok
test_v2_dynamic_segment_format (pandas.io.tests.test_ga.TestGoogle) ... ok
test_v3_advanced_segment_common_format (pandas.io.tests.test_ga.TestGoogle) ... ok
test_v3_advanced_segment_weird_format (pandas.io.tests.test_ga.TestGoogle) ... ok
test_v3_advanced_segment_with_underscore_format (pandas.io.tests.test_ga.TestGoogle) ... ok

----------------------------------------------------------------------
Ran 9 tests in 3.370s

OK
```

File: pandas/io/tests/test_ga.py [hunk 1]
```diff
@@ -142,7 +145,8 @@ def test_segment(self):
                 end_date=end_date,
                 segment=-2,
                 dimensions=['date', 'hour'],
-                parse_dates={'ts': ['date', 'hour']})
+                parse_dates={'ts': ['date', 'hour']},
+                index_col=0)
 
             assert isinstance(df, DataFrame)
```
Comment (jreback) [inline pandas/io/tests/test_ga.py]: can you fix these tests to use either:

``self.assertIsInstance(....)`` testing or

just compare vs an expected frame (e.g. ``assert_frame_equal``)

**Comment by parthea** [comment]
I've updated the tests as per the comments. The import error is gone in my latest commit. See job https://travis-ci.org/parthea/pandas/jobs/80171740

The new reason for skipped tests is `authentication error` as the tests require valid google credentials.
```
#16 pandas.io.tests.test_ga.TestGoogle.test_getdata: authentication error
#17 pandas.io.tests.test_ga.TestGoogle.test_iterator: authentication error
#18 pandas.io.tests.test_ga.TestGoogle.test_segment: authentication error
```

I've confirmed that the tests pass with valid credentials.

**Comment by jreback** [comment]
thanks!

**Pull request merged**